### DISCUSSION TITLE
GOCART look up tables

### DISCUSSION THREAD

**Discussion by gabersek** [NCAR/ccpp-scm#318] [Cases, Initial Conditions, Forcing] [answered]
I would like to enable GOCART aerosols interaction with radiation. The code is looking for 5 look up tables (e.g. 'optics_DU.dat' for dust). Any idea where to obtain them? Whom to contact?

Thank you in advance!

saša

**Comment by grantfirl** [comment, marked as answer]
Hi Saša,

We're going to include these as part of a separately downloadable asset from GitHub as part of the next release. In the meantime, I can provide these files to you. Email me at the email address associated with my GitHub user account, and I'll reply with the files attached. They are less than 2 MB in total, I think.

-Grant
